src/core/ui/grid/constants.ts

```diff
@@ -8,6 +11,19 @@ export const MAX_CONTENT_WIDTH_GUTTERS = 70;
 export const GRID_COLUMNS_MOBILE = 4;
 export const GRID_COLUMNS_DESKTOP = 12;
 
+export const useGlobalGridColumns = () => {
+  const isLarge = useMediaQuery<Theme>(theme => theme.breakpoints.up('md'));
+  const isSmall = useMediaQuery<Theme>(theme => theme.breakpoints.down('sm'));
```

Comment:
I think you're leaving SM out here (it's returning 8, maybe you did that on purpose... )
I mean, `up` is inclusive, `up('md')` will return true if MD or more. But `down` is exclusive so will only return true for XS.
If you have tested this or this was intended you can ignore this comment